chainer_chemistry/models/prediction/node_classifier.py

```diff
@@ -0,0 +1,44 @@
+from chainer import reporter
+from chainer_chemistry.models.prediction.classifier import Classifier
+from chainer_chemistry.dataset.graph_dataset.base_graph_data import PaddingGraphData  # NOQA
+
+
+class NodeClassifier(Classifier):
+
+    """A simple node classifier model.
+    """
+
+    def __call__(self, data, train_mask, valid_mask, *args, **kwargs):
+        """Computes the loss value for an input and label pair.
+        """
+        self.metrics = None
+        if isinstance(data, PaddingGraphData):
+            self.y = self.predictor(data.x, data.adj)
```

Comment:
maybe okay to use `self.predictor(data)` and wrap Chain side in SNS dataset example.

Reply:
fixed!